src/pages/projects.js

```diff
@@ -1,11 +1,32 @@
+import { PageHeader } from '../components/page-header/page-header'
+import { useState } from 'react'
 import { ProjectContent } from '../components/project-content/project-content'
 export const ProjectsPage = () => {
+    const [datas, setDatas] = useState([
+        {
+            title: "Göker Blog",
+            usedTechnology: "React, Css, HTML",
+            summery: "My first project where I shared the steps I took while becoming a developer.",
+            projectlink: "/",
+        },
+        {
+            title: "Göker Blog",
+            usedTechnology: "React, Css, HTML",
+            summery: "summurryfvfv summurryfvfv summurryfvfv summurryfvfv summurryfvfv",
+            projectlink: "/day2",
+        },
+        {
+            title: "Göker Blog",
+            usedTechnology: "React, Css, HTML",
+            summery: "summurryfvfv summurryfvfv summurryfvfv summurryfvfv summurryfvfv",
+            projectlink: "/day3",
+        },
+    ])
     return <div>
-        <h1>My projects</h1>
-        <ProjectContent title="Göker Blog" usedTechnology="React, Css, HTML" summery="summurryfvfv" projectlink="/day1"></ProjectContent>
-        <ProjectContent title="React: Very First Steps" usedTechnology="React" summery="summurryfvfv" projectlink="/day1"></ProjectContent>
-        <ProjectContent title="React: Very First Steps" usedTechnology="React" summery="summurryfvfv" projectlink="/day1"></ProjectContent>
-        <ProjectContent title="React: Very First Steps" usedTechnology="React" summery="summurryfvfv" projectlink="/day1"></ProjectContent>
+        <PageHeader>My projects</PageHeader>
+        {datas.map(data => (
+            <ProjectContent title={data.title} usedTechnology={data.usedTechnology} summery={data.summery} projectlink={data.projectlink}></ProjectContent>
```

Comment:
Where did you implement ProjectContent?

Reply:
this data is old. I changed it after you approved. my last commit is "link edit"

Reply:
Still the ProjectContent component needs to be somewhere which I can not see in this pr

Reply:
Anyways I found it it's already merged to main